### DISCUSSION TITLE
How to do this?

### DISCUSSION THREAD

**Discussion by FabioSilva8889** [mikacousin/olc#13] [Q&A] [answered]
Hi! My name is Fabio Silva. I'm from Fortaleza, Ceará - Brazil. Work with lighting for 8 years. I know QLC+ in the open source footprint. I found the idea interesting. I wanted to know how it uses, operates, records scenes, sequences...

**Comment by mikacousin** [comment, marked as answer]
Hello Fabio !
First of all, unlike QLC+, there are no devices, just dimmers.
The inspiration is in the family of AVAB consoles (or ETC Cobalt but very simplified). So no tracking.
You can use one sequence (more sequences are for chasers) composed with steps.
Each step use a memory (with channels levels) and times (in, out, delays, channels time, wait).
You can find info here : https://mikacousin.github.io/olc/
Keyboard shortcuts are directly in the program.

Hope this will help you.
